components/admin/content-preview.tsx

```
@@ -0,0 +1,80 @@
+      <Card>
+        <CardContent className="p-4">
+          <h3 className="text-lg font-medium mb-2">Frontmatter</h3>
+          <pre className="bg-muted p-4 rounded-md overflow-x-auto text-sm">{frontmatter}</pre>
+        </CardContent>
+      </Card>
+
+      <Card>
+        <CardContent className="p-4">
+          <h3 className="text-lg font-medium mb-2">Content Preview</h3>
+          <div className="prose dark:prose-invert max-w-none">
+            <div dangerouslySetInnerHTML={{ __html: renderedContent }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)